assets/src/components/form/layouts/two-column-admin/form-section.js

```diff
@@ -12,40 +12,63 @@ import FormInfo from './form-info';
 import { default as AutoFormRow } from '../../base/auto-form-row';
 import { default as AutoColumnRow } from './auto-column-row';
 
+const defaultFormInfo = {
+	requiredNotice: __(
+		'all fields marked with an asterisk (*) are required',
+		'event_espresso'
+	),
+	errorsNotice: __(
+		'please correct all form errors before proceeding',
```

Comment:
Sentences should start with Uppercase I think.

Reply:
i can do that... was intentionally doing so just as a style to avoid things looking too formal... but the tone of the text itself is pretty formal sounding already...

will change